src/routes/e2e/widget/inline/+page.svelte

```diff
@@ -32,31 +28,36 @@
       const response = await fetch(`${window.location.origin}/api/locale`);
       content = response.ok && (await response.json());
     }
-    // TODO: Add method to refresh form
-    new Widget({
-      target: formEl,
-      props: {
-        config: {
-          clientId: 'WebOAuthClient',
-          redirectUri: `${window.location.origin}/callback`,
-          scope: 'openid profile email me.read',
-          serverConfig: {
-            baseUrl: 'https://openam-crbrl-01.forgeblocks.com/am/',
-            timeout: 5000,
-          },
-          realmPath: 'alpha',
-        },
-        content,
-        links: {
-          termsAndConditions: 'https://www.forgerock.com/terms',
+
+    configuration().set({
+      config: {
+        clientId: 'WebOAuthClient',
+        redirectUri: `${window.location.origin}/callback`,
+        scope: 'openid profile email me.read',
+        serverConfig: {
+          baseUrl: 'https://openam-crbrl-01.forgeblocks.com/am/',
+          timeout: 5000,
         },
+        realmPath: 'alpha',
+      },
+      content,
+      links: {
+        termsAndConditions: 'https://www.forgerock.com/terms',
       },
     });
+
+    new Widget({ target: formEl });
+
     // Start the  journey after initialization or within the form.onMount event
-    journey.start({
+    const { start, subscribe } = journey();
+    start({
       journey: journeyParam || authIndexValue || undefined,
       resumeUrl: suspendedIdParam ? location.href : undefined,
     });
+    subscribe((event: any) => {
+      console.log(event);
+      userResponse = event?.user;
+    });
```

Comment:
Here's an example implementation of the new APIs. Notice how I configure the Widget, then instantiate the Widget, then grab the journey API, which returns `start` and `subscribe`. This does introduce a bit more surface area to the APIs, but creates much more consistency across all APIs. It's also much more scalable and flexible for future improvements.